interp/src/flatten/structures/environment/env.rs

```diff
@@ -721,6 +727,18 @@ impl<C: AsRef<Context> + Clone> Environment<C> {
         }
     }
 
+    pub fn print_pc_string(&self) {
+        let current_nodes = self.pc.iter();
+        let ctx = &self.ctx.as_ref();
+        for node in current_nodes {
```

Comment:
This `&` is unnecessary
```suggestion
        let ctx = self.ctx.as_ref();
```

Reply:
I have no idea why this ended up on the wrong line